documentation/modules/rn-2.2.adoc

```diff
@@ -94,3 +94,9 @@ The error status message for a VM with no operating system on the *Migration pla
 .Network, storage, and VM referenced by name in the Plan CR are not displayed in the web console.
 
 If a Plan CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-short} web console. The migration plan cannot be edited or duplicated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
+
+.A downloaded log archive file might include logs from a deleted resource
```

Comment:
Change to "Log archive file includes logs of a deleted migration plan or VM"

You don't need "might" in the header. You also don't need "downloaded". That can be explained in the description.